boom-app/src/helpers.ts

```diff
@@ -0,0 +1,55 @@
+import type { ObjectType } from './types'
+
```

Comment:
Gaaf! Door deze functies los te trekken is er een duidelijke plek waar je deze functies kan uitbreiden als we tzt ingewikkeldere json schemas gaan ondersteunen